.github/workflows/sync-back-to-dev.yml

```diff
@@ -0,0 +1,32 @@
+name: Sync Back to Development
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  sync-branches:
+    runs-on: ubuntu-latest
+    name: Syncing branches
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Node
```

Comment:
Unsure if setup-node is even needed, it shouldn't be needed AFAFICT.

Reply:
Did you try it? Its in the example on the actions repo.

Reply:
I didn't try it but it shouldn't need Node.js to be set up, AFAICT.

Reply:
Your hunch was correct:
https://github.com/pi-hole/land_of_confusion/runs/4772703213?check_suite_focus=true